examples/vl6180x_historytest.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2022 Jonas Schatz
+# SPDX-License-Identifier: MIT
+
+# Demo of reading the range from the history buffer of the VL6180x
+# distance sensor
+
+import time
+from typing import List
```

Comment:
I would remove the use of typing in the example, so it'll won't surprise people new to typing when they're using examples.  That and that `typing` isn't available on CircuitPython, so it'd need to be in a try/catch block similar to the library.

Reply:
Ah, indeed I was wondering why `typing` was in a try/except block in the main library, that explains it. Added your suggestions, thanks for the review.